.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [10.x, 12.x, 14.x]
```

Comment:
For Travis CI we also ran on "latest stable". I found https://github.com/actions/setup-node/issues/61#issuecomment-577548687 and it seems like you could use `*` for that, but it looks a bit strange and also I think it's enough that we just test against the currently supported LTS versions

Reply:
Agreed, it's not obvious that `*` means the latest stable. At least not until its documented (https://github.com/actions/setup-node/pull/104) :)

I think this is fine.